dbt_platform_helper/providers/platform_config_schema.py

```diff
@@ -8,15 +8,18 @@
 from schema import Schema
 from schema import SchemaError
 
+from dbt_platform_helper.constants import PLATFORM_CONFIG_SCHEMA_VERSION
+
 
 class PlatformConfigSchema:
     @staticmethod
     def schema() -> Schema:
         return Schema(
             {
+                "schema_version": PLATFORM_CONFIG_SCHEMA_VERSION,
                 "application": str,
                 Optional("deploy_repository"): str,
-                Optional("default_versions"): PlatformConfigSchema.__default_versions_schema(),
+                "default_versions": PlatformConfigSchema.__default_versions_schema(),
```

Comment:
Why is this required now?

Reply:
It was optional to make it backward compatible while we had the .platform-helper-version file support to fall back on. Makes sense to make it mandatory now that we are erroring and telling people to upgrade to 13 if they are using an older version.